boards/arm/nrf5340_dk_nrf5340/nrf5340_dk_nrf5340_cpuapp_partition_conf.dts

```diff
@@ -39,16 +39,22 @@
 
 /* Default SRAM planning when building for nRF5340 with
  * ARM TrustZone-M support
- * - Lowest 64 kB SRAM allocated to Secure image (sram0)
- * - Middle 64 kB SRAM allocated as Shared memory (sram0_shared)
+ * - Lowest 64 kB SRAM allocated to Secure image (sram0_s)
+ * - Middle 384 kB allocated to Non-Secure image (sram0_ns)
+ * - Upper 64 kB SRAM allocated as Shared memory (sram0_shared)
  *   (see nrf5340_dk_nrf5340_shared_sram_planning_conf.dts)
- * - Upper 384 kB allocated to Non-Secure image (sram0_ns)
  */
-
 &sram0 {
-	reg = <0x20000000 DT_SIZE_K(64)>;
+	reg = <0x20000000 DT_SIZE_K(448)>;
+};
+
+&sram0_s {
+	reg = <0x20000000 0x10000>;
```

Comment:
Why sizes of `sram0` and `sram0_shared` are specified with the `DT_SIZE_K` macro, and for `sram0_s` and `sram0_ns` with hex values? It looks a bit inconsistent.

Reply:
That's because _those_ nodes are actually partitions, @anangl . So they are written as the slot0, slot1, etc. paritions for flash....

Reply:
> That's because _those_ nodes are actually partitions, @anangl . So they are written as the slot0, slot1, etc. paritions for flash....

That's what I assumed. But what about `sram0_shared`?

Reply:
That memory region is outside the partitioning of the Application MCU SRAM: it's definition needs to lie in a place that is visible, also, from the Network MCU, because I don't want to have that re-defined as a DTS node for the Network core (CPUNET). Practically, this memory "belongs" to both MCUs. Since - from DTS perspective - it is not a partition, we have defined it as a normal RAM node. This PR only changes it's reg properties (start address, size).